packages/react-core/src/components/Menu/MenuContainer.tsx

```diff
@@ -55,10 +58,11 @@ export const MenuContainer: React.FunctionComponent<MenuContainerProps> = ({
   toggle,
   toggleRef,
   onOpenChange,
+  onToggleArrowKeydown,
   zIndex = 9999,
   popperProps,
   onOpenChangeKeys = ['Escape', 'Tab'],
-  shouldFocusFirstItemOnOpen = true,
+  shouldFocusFirstItemOnOpen = false,
```

Comment:
This is breaking changing the default behavior, and we're post v6 at this point, but I'd be fine still making this change since it can be considered a "bug" in its inconsistency with the other menus. I'm not sure how many people are using `MenuContainer` directly opposed to Select/Dropdown, but I believe there are at least a few.

Just wanted to point it out in case others think differently.

Reply:
@tlabaj

Reply:
I think this is fine to change here as a bug fix.